### PR TITLE
Allow Lab_Rats to become tamable

### PR DESCRIPTION
**Pull request by LordBarkBread**
#### Summary 

Content "Allow Lab Rats to become tamable"

Feature.

#### Purpose of change 
Lab rats are already a form of domesticated as most if not all have been bred and raised within captivity of labs, giving them some food should tame them as a pet to follow you, while they aren't "the best pets" they are still used to human contact and from being around scientists/researchers.

#### Describe the solution
Adding the strings of "petfood" "pet" and the "flags" "CANPLAY".

#### Testing
Taming a lab rat is very much just like taming other animals within the game, naming it and petting/playing with it proceed the same results. 
image

### CODE REVIEW

**Comment by friendfrog** [comment]
I see a typo here, it says "It lets out a happy squeak **as runs up** to your foot" instead of "as it runs up"
I don't actually see that line in your changes though so ¯\_(ツ)_/¯

**Comment by LordBarkBread** [comment]
> I see a typo here, it says "It lets out a happy squeak **as runs up** to your foot" instead of "as it runs up" I don't actually see that line in your changes though so ¯_(ツ)_/¯
> 
> EDIT: I see you already changed it to something different than the sample image. Ignore me!

Oh that is 100% my fault, I wasn't paying enough attention it seems. ^^;

File: data/json/monsters/mammal.json [hunk 1]
```diff
@@ -2542,3 +2544,4 @@
     "upgrades": { "age_grow": 500, "into": "mon_reindeer" }
   }
 ]
+
```
Comment (Maleclypse) [inline data/json/monsters/mammal.json:2547]:
```suggestion
```

File: data/json/monsters/mammal.json [hunk 2]
```diff
@@ -197,7 +197,9 @@
     "upgrades": { "half_life": 13, "into_group": "GROUP_RATKIN_EVOLVED" },
     "fear_triggers": [ "PLAYER_CLOSE", "SOUND" ],
     "special_attacks": [ [ "EAT_FOOD", 120 ] ],
-    "flags": [ "SEES", "SMELLS", "HEARS", "KEENNOSE", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
+    "petfood": { "food": [ "DOGFOOD" ], "feed": "The %s seems to like you!  It lets you pat its head and seems friendly." },
+    "pet": "The %s lets out a happy squeak as you give its head a small scritch.",
+    "flags": [ "SEES", "SMELLS", "HEARS", "KEENNOSE", "CANPLAY", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
```
Comment (dseguin) [inline data/json/monsters/mammal.json:202]:
```suggestion
    "petfood": {
      "food": [ "DOGFOOD" ],
      "feed": "The %s seems to like you!  It lets you pat its head and seems friendly.",
      "pet": "The %s lets out a happy squeak as you give its head a small scritch."
    },
    "flags": [ "SEES", "SMELLS", "HEARS", "KEENNOSE", "CANPLAY", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
```

Just a small correction, moving the `"pet"` field inside the `"petfood"` object. Tested in-game and it works (hopefully this resolves the stalled test).

**Pull request merged**